src/main/java/org/radarbase/appserver/service/FcmNotificationService.java

```diff
@@ -283,7 +283,6 @@ public FcmNotifications scheduleAllUserNotifications(String subjectId, String pr
     @Transactional
     public FcmNotificationDto scheduleNotification(String subjectId, String projectId, long notificationId) {
 
-        User user = subjectAndProjectExistElseThrow(subjectId, projectId);
         Notification notification = this.getNotificationByProjectIdAndSubjectIdAndNotificationId(projectId, subjectId, notificationId);
```

Comment:
what happens if the notification does not exist? Please check and throw Not Found Exception

Reply:
Currently, it is throwing ```InvalidNotificationDetailsException```  and returning ```EXPECTATION_FAILED```. I will change it to return 404 by throwing ```NotFoundException```

Reply:
ok, that is fine I guess for now. You can leave it as it is. Sorry, I thought this was a function of the repository. My bad.